R/check_documentation.R

```diff
@@ -1,4 +1,39 @@
 #' Check the documentation
+#'
+#' Checks if all required documentation is present and rendered.
+#' A common problem is that one forgets to render the documentation after
+#' updating it.
+#' You'll get an error when you forgot to update the documentation.
+#' But the function will render the documentation while checking it.
+#' So the rendered documentation is available after the check completes.
+#' The function relies on git to detect changes.
+#' Run the checks after committing all changes.
+#' You can amend the commit in case you forgot to render something.
```

Comment:
I find this part very hard to read: it somehow seems to be a set of unrelated sentences, but when looking closely at the content and making the puzzle, I discovered together they are trying to explain what the function does, how the user can deal with it, and what the function relies on to detect the problem. Honestly, I find hard to pinpoint what exactly is wrong, and I do not feel competent to rewrite it.  Maybe it has something to do with finding the appropriate words to connect sentences?  Maybe it could help as well to put all sentences on what the function does first, and the sentences on how the user can deal with it afterwards?  And maybe the sentence `The function relies on git to detect changes.` could be added after `Checks if all required documentation is present and rendered.` (1st sentence of documentation) by using the conjunction `by`?